ui/StatusQ/src/StatusQ/Controls/StatusBaseButton.qml

```diff
@@ -33,6 +34,8 @@ Button {
     property color textColor
     property color disabledTextColor
     property color borderColor: "transparent"
+    property int textAlignment: Qt.AlignVCenter | Qt.AlignHCenter
+    property bool textFillWidth: false
```

Comment:
These should not be needed; the `contentItem` is a RowLayout and as such should accomodate